operators/bosh-operator/DirectorService.js

```diff
@@ -237,18 +247,22 @@ class DirectorService extends BaseDirectorService {
     };
     return this
       .initialize(operation)
+      .catch(err => logger.error(`Error occurred while finding network index for update
```

Comment:
Just a question , in case of update why do we need to fail in case network segment index determination fails via BOSH ? Since during update the deployment name is already present , we can get it from the same , right ?

Reply:
Here we are not failing, staggered. Deployment name is figured out from local cache. If cache miss happens we go to bosh and figure out. So, if bosh calls fails here, staggered.

But one error handling missed here, ServiceInstanceNotFound, thanks for giving a comment here 👍 . would update.